nixos/modules/system/boot/loader/systemd-boot/systemd-boot-builder.py

```diff
@@ -162,7 +162,8 @@ def remove_old_entries(gens: List[Tuple[Optional[str], int]]) -> None:
             else:
                 prof = "system"
```

Comment:
Should we than not just set this to None?
```suggestion
                prof = None
```